CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ Bug Fixes
   timely manner. Now, WebSocket timeouts can be identified in around 30 seconds.
   If a WebSocket timeout occurs, for example, due to a WebSocket disconnect,
   Room will emit a "disconnected" event with SignalingConnectionTimeoutError.
+- If an ICE failure occurs, for example due to disconnecting a VPN, but the
+  but the signaling connection remains online, twilio-video.js will attempt an
```

Comment:
@markandrus Extra `but the` at the beginning of this line?

Also this can be rewritten as: `... connection remains online, an ICE restart will be attempted to repair the media connection.`?

Reply:
Oh, nice catch. I'll try your re-wording, too.

Reply:
I fixed the duplicate "but the", but, after reviewing, I kept the second half as-is, since I think it is more active voice (whereas "will be attempted" is more passive voice).